test_themes/tests/test_crawl.py

```diff
@@ -49,4 +49,11 @@ def test_02_homepage_tour_every_theme(self):
         Website = self.env['website']
         websites_themes = Website.get_test_themes_websites()
         for website in websites_themes:
+            # TODO: remove this invalidation and invalidation in theme feature.
+            # They are missing invalidations of template ormcache and others.
+            # The configurator_apply method and various methods used for theme
+            # added on `ir.module.module` from website write directely on
+            # `ir.model.data` and update attachments, views, xmlids.
+            self.env.registry.clear_cache('templates')
```

Comment:
TBH I am not sure to understand: how is what you describe related to starting tours? Is not there a bug/something to do in the community code? 🤔

Reply:
Indeed, there's a change to be made in the community theme section, but the code is very difficult to understand, and the number of places where xmlid style records are modified without invalidating the corresponding models is significant. I've asked for help understanding the system, but the fix can only be done by those who developed and modified this code (in community and themes).

The cache errors are already present. The higher-level cache reveals an invalidation bug by accumulating theme installations with template and attachment modifications.